projects/plugins/jetpack/_inc/polldaddy-shortcode.js

```diff
@@ -1,74 +1,94 @@
-( function ( $ ) {
-	window.polldaddyshortcode = {
+( function ( w, d ) {
+	w.polldaddyshortcode = {
 		render: function () {
-			var ratings = $( 'div.pd-rating[data-settings]' );
-			var polls = $( 'div.PDS_Poll[data-settings]' );
+			const ratings = d.querySelectorAll( 'div.pd-rating[data-settings]' );
+			const polls = d.querySelectorAll( 'div.PDS_Poll[data-settings]' );
 
-			if ( polls ) {
-				$.each( polls, function () {
-					var poll = $( this ).data( 'settings' );
+			polls.forEach( pollEl => {
+				const isInitialized = pollEl.getAttribute( 'data-pd-init-done' );
```

Comment:
I wonder whether `hasAttribute` would make more sense here. We don't seem to care about the value.

Reply:
Good suggestion 👍